hawkbit-rest/hawkbit-ddi-resource/src/main/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootController.java

```diff
@@ -313,6 +313,14 @@ private static boolean checkModule(final String fileName, final SoftwareModule m
         return ResponseEntity.notFound().build();
     }
 
+    private HandlingType calculateDownloadType(final Action action) {
+        if (action.isDownloadOnly() || action.isForce()) {
+            return HandlingType.FORCED;
+        } else {
```

Comment:
else could be removed, or use the `? :` operator:
`return action.isDownloadOnly() || action.isForce() ? HandlingType.FORCED : HandlingType.ATTEMPT;`